js/commonjs/rewrite_tests_for_commonjs.js

```diff
@@ -67,7 +67,7 @@ console.log("var global = Function('return this')();");
 console.log("var googleProtobuf = require('google-protobuf');");
 console.log("var testdeps = require('testdeps_commonjs');");
 console.log("global.goog = testdeps.goog;");
-console.log("global.jspb = testdeps.jspb;");
+console.log("global.jspb = googleProtobuf.jspb;");
```

Comment:
See comment above. We should use `google-protobuf`'s jspb instead of the one we exported for tests.